spec.html

```diff
@@ -35770,6 +35770,62 @@ <h1>Array.prototype.findIndex ( _predicate_ [ , _thisArg_ ] )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-array.prototype.findlast">
+        <h1>Array.prototype.findLast ( _predicate_ [ , _thisArg_ ] )</h1>
+        <emu-note>
+          <p>_predicate_ should be a function that accepts three arguments and returns a value that is coercible to a Boolean value. `findLast` calls _predicate_ once for each element of the array, in descending order, until it finds one where _predicate_ returns *true*. If such an element is found, `findLast` immediately returns that element value. Otherwise, `findLast` returns *undefined*.</p>
```

Comment:
> in descending order

I think that either "in reverse order" or "in descending index order" would be clearer here.

Reply:
```suggestion
          <p>_predicate_ should be a function that accepts three arguments and returns a value that is coercible to a Boolean value. `findLast` calls _predicate_ once for each element of the array, in descending order, until it finds one where _predicate_ returns *true*. If such an element is found, `findLast` immediately returns that element. Otherwise, `findLast` returns *undefined*.</p>
```

"element" is clear enough on its own.

Reply:
Done.